packages/story-editor/src/app/api/apiProvider.js

```diff
@@ -53,9 +53,11 @@ const STORY_FIELDS = [
   'permalink_template',
   'style_presets',
   'password',
+  'story-tags',
```

Comment:
@swissspidy @spacedmonkey I think we're fine without this since all associated terms should be in the `_embed` . Is it fine just to use the `_embed['wp:term']` as the reference for all associated terms?

<img width="746" alt="Screen Shot 2021-09-13 at 12 01 31 PM" src="https://user-images.githubusercontent.com/35983235/133126330-6c692994-a54e-4999-87ad-c6143ce3caf5.png">

Reply:
Yes